.github/workflows/release.yml

```
@@ -150,14 +151,29 @@ jobs:
           git commit -am "docs: release ${{ needs.process-inputs.outputs.MAJOR_MINOR }}"
           # Clean up auxiliary files, so next steps run on a clean tree
           git clean -fx :/
-      - name: Download updated contrast releases file (from release branch)
+      - name: Download artifacts
         uses: actions/download-artifact@65a9edc5881444af0b9093a5e628f2fe47ea3b2e #v4.1.7
         with:
-          name: contrast-releases.json
-          path: ./contrast-main/packages/contrast-releases.json
+          merge-multiple: true
+      - name: Patch latest references in docs
+        shell: bash
+        run: |
+          while IFS= read -r line; do
+              target=$(echo "$line" | cut -d"=" -f2)
+              source=$(echo "$line" | sed "s/:.*//" | sed "s/\./\\\./g")
+              find ./contrast-main/docs/versioned_docs/version-$MAJOR_MINOR -type f -exec sed -i "s#$source\(:[a-zA-Z0-9_.-]\{1,\}\(@sha256:[a-fA-F0-9]\{64\}\)\?\|@sha256:[a-fA-F0-9]\{64\}\)#$target#g" {} \;
+          done <"./image-replacements.txt"
+
+          find ./contrast-main/docs/versioned_docs/version-$MAJOR_MINOR -type f -exec sed -i "s#github.com/edgelesssys/contrast/releases/latest/download/#github.com/edgelesssys/contrast/releases/download/${{ inputs.version }}/#g" {} \;
+      - name: Commit updated docs
+        working-directory: contrast-main
+        run: |
+          git add docs/versioned_docs/version-$MAJOR_MINOR
+          git commit -m "docs: patch latest references"
```

```suggestion
          git commit -m "docs: update release download urls"
```
Might be a bit clearer?